src/codegeneration/ModuleTransformer.js

```diff
@@ -276,8 +276,8 @@ export class ModuleTransformer extends TempVarTransformer {
     assert(this.moduleName);
     let name = tree.token.processedValue;
     // import/module {x} from './name.js' is relative to the current file.
-    let normalizedName = System.normalize(name, this.moduleName);
-    return parseExpression `System.get(${normalizedName})`;
+    let normalizedName = $traceurRuntime.normalizeModuleName(name, this.moduleName);
```

Comment:
Yes we think of this as the difference between a `build` and a `bundle` in SystemJS. A `build` can use a loader internally, but it just doesn't expose the loader globally.